src/routes/signin/webauthn.ts

```diff
@@ -119,15 +117,15 @@ export const signInVerifyWebauthnHandler: RequestHandler<
     return sendError(res, 'invalid-request');
   }
 
-  const userAuthenticators = await gqlSdk
+  const authenticator = await gqlSdk
     .getUserAuthenticators({
       id: user.id,
     })
-    .then((gqlres) => gqlres.authUserAuthenticators);
-
-  const authenticator = userAuthenticators.find(
-    (auth) => auth.credentialId === credential.id
-  );
+    .then(({ authUserAuthenticators }) =>
+      authUserAuthenticators.find(
+        ({ credentialId }) => credentialId === credential.id
+      )
+    );
```

Comment:
I would try not to do anything in callback functions (`.then()`) and instead always follow the same `async`/`await` pattern.

Something like this:

```js
const { authUserAuthenticators } = await gqlSdk
  .getUserAuthenticators({
    id: user.id,
  })

const authenticator = authUserAuthenticators.find(
  (auth) => auth.credentialId === credential.id
);